django_sorcery/db/models.py

```diff
@@ -423,3 +424,29 @@ def full_clean_flush_handler(session, **kwargs):
     for i in session.dirty | session.new:
         if isinstance(i, Base):
             i.full_clean()
+
+
+def force_auto_coerce(cls=None):
+
+    m = cls.__mapper__ if cls else sa.orm.mapper
+
+    from django_sorcery.field_mapping import get_field_mapper
+
+    @sa.event.listens_for(m, "mapper_configured")
+    def hook_coercers(mapper, class_):
+        field_mapper = get_field_mapper()(model=class_)
+        info = model_info(class_)
+
+        def _coerce(target, value, oldvalue, initiator, form_field):
+            return form_field.clean(value)
+
+        for name, prop in info.properties.items():
+
+            form_field = field_mapper.build_field(prop, required=False, localize=True)
+
+            if form_field is None:
+                continue
+
+            sa.event.listen(getattr(class_, name), "set", partial(_coerce, form_field=form_field), retval=True)
```

Comment:
its possible this can be added multiple times. for example when somebody manually calls `configure_mappers` multiple times for whatever reason. might make sense memoize which props are coersed already not to add duplicate signals

Reply:
added a partial memo